apps/e2e/ag-grid-storybook/src/app/data-manager/data-manager.component.ts

```diff
@@ -76,7 +76,7 @@ export class DataManagerComponent implements OnInit {
   public viewId = 'gridView';
 
   public dataState: SkyDataManagerState;
-  public items = data;
+  public items = data.slice(0, 50);
```

Comment:
This file has small changes to help simplify rendering. These tests will also now use Cypress screenshots.